cypress/integration/collections_spec.js

```diff
@@ -172,17 +171,14 @@ describe('Dashboard Collections Page', () => {
       // collections with which to populate the dropdown on the collection
       // details page.
       cy.visit('/collections');
-      // TODO [DOP, 2020-05-14] Workaround until CUMULUS-1996 is resolved
-      // Stop timer so that it does not dispatch listCollections and change
-      // the order of items in the list after date is cleared
-      cy.get('.form__element__updateToggle .form__element__clickable').click();
-      cy.clearStartDateTime();
       cy.wait('@getCollections');
       cy.get('.table .tbody .tr').should('have.length', 5);
 
       cy.contains('.table .tbody .tr a', name)
-        .should('have.attr', 'href', `/collections/collection/${name}/${version}`)
-        .click();
+        .then(($res) => {
+          expect($res).to.have.attr('href', `/collections/collection/${name}/${version}`);
+          cy.wrap($res).click();
+        });
```

Comment:
I'm hoping this means that the same item will be checked for the href and then will be clicked. :shrug: